api4/schema.graphqls

```diff
@@ -70,21 +70,22 @@ type Channel {
 }
 
 type ChannelMember {
-	channel          : Channel
-	user             : User
-	roles            : [Role]!
-	lastViewedAt     : Float!
-	msgCount         : Float!
-	mentionCount     : Float!
-	mentionCountRoot : Float!
-	msgCountRoot     : Float!
-	notifyProps      : StringMap!
-	lastUpdateAt     : Float!
-	schemeGuest      : Boolean!
-	schemeUser       : Boolean!
-	schemeAdmin      : Boolean!
-	explicitRoles    : String!
-	cursor: String
+	channel           : Channel
+	user              : User
+	roles             : [Role]!
+	lastViewedAt      : Float!
+	msgCount          : Float!
+	mentionCount      : Float!
+	urgentMentionCount: Float!
```

Comment:
Lovely to see the gql schema extended. Please also add a test for the existence of the field in api4/resolver_channel_member_test.go. Thanks!